src/frontend/src/components/common/Modal/_modal.scss

```diff
@@ -5,17 +5,47 @@
   width: 100%;
   height: 100%;
   background: rgba(0, 0, 0, 0.6);
+  border-radius: 5px;
 
+  .modal-title {
+    display: block;
+    font-weight: bold;
+    position: relative;
+    text-align: center;
+    padding-top: 1em;
+    color: black;
+  }
+
+  .horizontal-divider {
+    width: 100%;
+    height: 0;
+    border: 2px solid #E2A014;
+    margin-block: 18px 16px;
+  }
+
+   .btn-secondary {
+    margin-top: 0.2em;
+    border: 0;
+   }
```

Comment:
This affects every button with this class so I suggest nesting a class in the btn-secondary class inside of  _buttons.scss. 

Implementation would look something like 
```
.btn-secondary {
  background-color: $bg-white;
  color: $primary-blue;
  border: 2px solid $primary-blue;
  border-radius: 4px;
  text-align: center;
  padding-block: 12px;
  padding-inline: 25px;
  text-decoration: none;
  display: flex;
  max-width: fit-content;
  white-space: nowrap;
  gap: 0.7em;
  align-items: center;
  justify-content: space-evenly;
  font-weight: 700;
  letter-spacing: 1px;
  cursor: pointer;

  & > .icon {
    font-size: x-large;
  }

  &:hover {
    text-decoration: underline;
    opacity: 0.8;
  }

  &:focus {
    outline: 4px solid $accent-lightblue;
    outline-offset: 1px;
  }

  &:active {
    opacity: 1;
  }

  &.btn-secondary__modal {
     margin-top: 0.2em;
     border: 0;
   }
}
```

Reply:
done